bedrock/settings/base.py

```diff
@@ -596,31 +596,31 @@ def facebook_tab_url_lazy():
     'cs': 'https://wiki.mozilla.org/Mobilizer/MobilizerCzechRepublic/'
 }
 
-DONATE_SPANISH_LINK = ('https://donate.mozilla.org/es/?presets=100,50,25,15&amount=50'
+DONATE_SPANISH_LINK = ('https://donate.mozilla.org/es/?amount=50'
     '&ref=EOYFR2015&utm_campaign=EOYFR2015&utm_source=mozilla.org'
     '&utm_medium=referral&utm_content={source}&currency=eur')
 
 DONATE_LOCALE_LINK = {
     'default': (
-        'https://sendto.mozilla.org/page/contribute/Give-Now?source={source}'
+        'https://donate.mozilla.org/en-US/?source={source}'
```

Comment:
I actually agree and was thinking that, because when they do get the locale switching implemented we will not be forcing en-US with the default. Because there was a note added to always add a locale I thought it best to leave it in. With that said, it will currently fallback to en-US anyhow so.... I reckon dropping that off is fine.